hasher-matcher-actioner/terraform/main.tf

```diff
@@ -125,3 +125,25 @@ resource "aws_sqs_queue_policy" "pdq_hasher_queue" {
   queue_url = aws_sqs_queue.pdq_images_queue.id
   policy    = data.aws_iam_policy_document.pdq_hasher_queue.json
 }
+
+
+# Connect Hashing Data to API
+
+module "api_surface" {
```

Comment:
If we go with just `api` then probably change this, too (though you've probably already dealt with this...)